<launch>

  <arg name="port" default="/dev/ttyUSB0"/>

  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="$(arg port)"/>
    <param name="baud" value="115200"/>
  </node>
</launch>
